app/controllers/users/two_factor_authentication/setup_controller.rb

```diff
@@ -51,7 +51,8 @@ def qr_code_uri
 private
 
   def save_two_factor_method(method)
-    # TODO: This will be implemented in future.
+    current_user.second_factor_method = method
+    current_user.save
```

Comment:
use `current_user.update_attributes(second_factor_method: method)` to save yourself the save (sic).